.circleci/config.yml

```diff
@@ -29,6 +29,15 @@ jobs:
     working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
     steps:
       - checkout
+      - run:
+          name: Build plugin artifact for tests
+          working_directory: /tmp
+          command: |
+            go version
+            export GOMAXPROCS=2
+            export PATH=$GOPATH/bin:$PATH
+            go get -d github.com/andremedeiros/leftpad
+            go build -buildmode=plugin -o /tmp/leftpad.so github.com/andremedeiros/leftpad
```

Comment:
I do think that:
1. the test should work without an additional step before that of building the plugin ... this should probably happen in the test ?
2. I would really prefer if the test plugin is in our repo - probably in the `samples` folder? (great plugin name ;) )

Reply:
> great plugin name

I can't take the credit. Someone in the slack suggested that for a test plugin, and I just went with it :-P